src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -1,14 +1,15 @@
 package org.xstefank.webhook;
 
-import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
Same, nothing bad, but please understand that it is not a part of the fix for this issue :) and it's only polluting the PR with non-relevant information (this is again IDE setting, you should be aware of what code are you changing)